### PR TITLE
Document how to release tsickle

### PR DESCRIPTION
**Pull request by mprobst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,3 +119,10 @@ rewrite the golden files when you run it.
 
 Export the environment variable `TEST_FILTER`, a regex, to limit the end-to-end
 tests (found in `test_files/...`) run tests with a name matching the regex.
+
+### Releasing
+
+On a new branch, run `npm version <major|minor|patch|...> -m 'rel: %s'` (see
+`npm help version` for details) to update the version in `package.json`, commit
+the changes, and create a git tag. Push the branch, get it reviewed and merged,
```
Comment (evmar) [inline README.md]: "create a git tag like v1.2.3" (or is it just "1.2.3" or "release-1.2.3"?)

File: README.md [hunk 2]
```diff
@@ -119,3 +119,10 @@ rewrite the golden files when you run it.
...
+then run `npm publish` from the master branch.
```
Comment (evmar) [inline README.md]: What credentials are needed here?

File: README.md (hunk 1 above)
Comment (mprobst) [reply README.md]: It'll create whatever it creates, we don't have influence on it and we don't really care either, do we?

File: README.md (hunk 2 above)
Comment (mprobst) [reply README.md]: done.

File: README.md (hunk 1 above)
Comment (evmar) [reply README.md]: Ah I read it as
on a new branch
1) run npm version
2) commit the changes
3) create a git tag

but I now see your parse too.

**Pull request merged**